2.0/README.md

```diff
@@ -0,0 +1,500 @@
+* ClosePath
+
+```
+Encoded as: [ 9 6 12 18 10 12 24 44 15 ]
+              |       |              `> [00001 111] command id 7 (ClosePath), length 1
+              |       |       ===== relative LineTo(+12, +22) == Line to Point (20, 34)
+              |       | ===== relative LineTo(+5, +6) == Line to Point (8, 12)
+              |       `> [00010 010] = command id 2 (LineTo), length 2
+              | ==== relative MoveTo(+3, +6)
+              `> [00001 001] = command id 1 (MoveTo), length 1
+```
+
+##### 4.3.5.6. Example Multi Polygon
+
+An example of a more complex encoding of two polygons, one with a hole. The position of the points for the polygons are shown below. The winding order of the polygons is VERY important in this example as it signifies the difference between interior rings and a new polygon.
+
+* Polygon 1:
+  * Exterior Ring:
+    * (0,0)
+    * (10,0)
+    * (10,10)
+    * (0,10)
+    * (0,0) *Path Closing as Last Point*
+* Polygon 2:
+  * Exterior Ring:
+    * (11,11)
+    * (20,11)
+    * (20,20)
+    * (11,20)
+    * (11,11) *Path Closing as Last Point*
+  * Interior Ring:
+    * (13,13)
+    * (13,17)
+    * (17,17)
+    * (17,13)
+    * (13,13) *Path Closing as Last Point*
+
+This polygon would be encoded with the following set of commands:
+
+* MoveTo(+0,+0)
+* LineTo(+10,+0)
+* LineTo(+0,+10)
+* LineTo(-10,+0) // Cursor at 0,10 after this command
+* ClosePath // End of Polygon 1
+* MoveTo(+11,+1) // NOTE THAT THIS IS RELATIVE TO LAST LINETO!
+* LineTo(+9,+0)
+* LineTo(+0,+9)
+* LineTo(-9,+0) // Cursor at 11,20 after this command
+* ClosePath // This is a new polygon because area is positive!
+* MoveTo(+2,-7) // NOTE THAT THIS IS RELATIVE TO LAST LINETO!
+* LineTo(+0,+4)
+* LineTo(+4,+0)
+* LineTo(+0,-4) // Cursor at 17,13
+* ClosePath // This is an interior ring because area is negative!
+
+### 4.4. Raster Encoding
+
+Currently the use of `raster` within features is considered **experimental**. Encoders MAY support this field, but are not required to do so at this time. In future releases of this specification a more detailed set of requirements for `raster` support may be defined. It is likely in the future that a format or set of formats for images or rasters will be specified as supported for this field.
+
+### 4.5. Feature Attributes
+
+Feature attributes are encoded as pairs of integers in the `tag` field of a feature. The first integer in each pair represents the index of the key in the `keys` set of the `layer` to which the feature belongs. The second integer in each pair represents the index of the value in the `values` set of the `layer` to which the feature belongs. Every key index MUST be unique within that feature such that no other attribute pair within that feature has the same key index. A feature MUST have an even number of `tag` fields. A feature `tag` field MUST NOT contain a key index or value index greater than the number of elements in the layer's `keys` or `values` set, respectively.
```

Comment:
`The keys include all the keys of features used in the layer, and each key may be referenced by its positional index in this set of keys, with the first key having an index of 0.`

It is mentioned earlier in the spec as being 0 indexed so the wording is correct I believe @mourner